karapace/schema_registry_apis.py

```diff
@@ -839,8 +836,11 @@ def write_new_schema_local(self, subject, body, content_type):
 
             for old_version in check_against:
                 old_schema = subject_data["schemas"][old_version]["schema"]
+                validated_old_schema = ValidatedTypedSchema.parse(
```

Comment:
At this point an error can be raised if the previously stored schema cannot be parsed. This is possible when upgrading Avro library, for example version 1.10 allows enum to have a default that is not in the symbol set. Forthcoming 1.11 would raise an error.